src/backgroundblurprocessor/BackgroundBlurProcessorProvided.ts

```diff
@@ -47,6 +47,20 @@ export default class BackgroundBlurProcessorProvided
     this.logger.info('BackgroundBlur processor successfully created');
     this.logger.info(`BackgroundBlur spec: ${this.stringify(this.spec)}`);
     this.logger.info(`BackgroundBlur options: ${this.stringify(options)}`);
+
+    screen.orientation.addEventListener('change', _event => {
```

Comment:
Can this be in `BackgroundFilterProcessor`?

Reply:
What was the issue?

Reply:
It was not changing the orientation

Reply:
Is the listener not called? I don't see how it wouldn't work. At the very least can you put the listener function in shared code?

Reply:
The listener is called, but sometimes there is a delay which causes the canvas context to not change and hence causing the video to be garbled